waku/v2/node/wakunode2.go

```diff
@@ -236,6 +236,7 @@ func New(opts ...WakuNodeOption) (*WakuNode, error) {
 			}()
 			return r
 		},
+		autorelay.WithBootDelay(2*time.Second),
```

Comment:
By default, go-libp2p will wait 3 minutes before negotiating circuit relay addresses. See that function documentation to understand the reason behind that!